projects/kit/components/textarea/textarea.template.html

```diff
@@ -12,7 +12,7 @@
 >
     <div
         *ngIf="hasCounter"
-        automation-id="tui-text-area__counter"
+        automation-id="tui-textarea__counter"
```

Comment:
Can we exempt `automation-id` from migration and leave them as they are?

Reply:
i'm not sure if it's easy